software/spmd/bsg_cuda_lite_runtime/float_vec_add/Makefile

```diff
@@ -0,0 +1,53 @@
+#########################################################
+# Network Configutaion
+# If not configured, Will use default Values
+#	bsg_global_X ?= $(bsg_tiles_X)
+#	bsg_global_Y ?= $(bsg_tiles_Y)+1
+
+bsg_global_X = 4
```

Comment:
shouldn't these be `?=`, not `=`?

Reply:
@bornaehsani

Reply:
I've been using the same Makefile for all cuda lite runtime regression tests since the dawn of time. Any issues here is for all regression tests. Maybe it's time for a refactoring of these Makefiles. We can discuss when you get back.